### PR TITLE
fix: convert rgba() to rgb()

### LINKED ISSUE #164: Modern JS: replaces occurrences of `rgba` and `hsla`
There are quite a few occurrences of `rgba` and `hsla` in the repo.

We should only use the modern syntax of `rgb` and `hsl`. (No need to update the tools directory, they would need much more work if we want to keep them…)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fix https://github.com/mdn/css-examples/issues/164

Regex: `rgba\((\d\d?\d?%?), ?(\d\d?\d?%?), ?(\d\d?\d?%?), ?(..?.+)\)`
Replace: `rgb($1 $2 $3 / $4)`

An update to the new color function syntax has been done in [a separate PR](https://github.com/mdn/css-examples/pull/196).

### CODE REVIEW

File: flexbox/use-cases/input-button.html [hunk 1]
```diff
@@ -75,11 +75,11 @@
         line-height: 1;
       }
       .wrapper input[type="text"] {
-        background-color: rgba(96, 139, 168, 0.5);
+        background-color: rgb(96 139 168 / 0.5);
         border-right: 1px solid rgb(96, 139, 168);
       }
       .wrapper input[type="submit"] {
-        background-color: rgba(96, 139, 168, 1);
+        background-color: rgb(96 139 168 /  1);
```
Comment (bsmth) [inline flexbox/use-cases/input-button.html:82]: What about:

```suggestion
        background-color: rgb(96 139 168);
```

File: flexbox/use-cases/label-input-button.html [hunk 2]
```diff
@@ -76,11 +76,11 @@
         line-height: 1;
       }
       .wrapper > input[type="text"] {
-        background-color: rgba(96, 139, 168, 0.5);
+        background-color: rgb(96 139 168 / 0.5);
         border-right: 1px solid rgb(96, 139, 168);
       }
       .wrapper input[type="submit"] {
-        background-color: rgba(96, 139, 168, 1);
+        background-color: rgb(96 139 168 /  1);
```
Comment (bsmth) [inline flexbox/use-cases/label-input-button.html:83]: Same here, should we convert all with `r g b / 1` to `r g b`?

File: houdini/css_painting_api/example-boxbg.html [hunk 3]
```diff
@@ -7,16 +7,16 @@
     <style>
       li {
         background-image: paint(boxbg);
-        --boxColor: hsla(55, 90%, 60%, 1);
+        --boxColor: hsl(55 90% 60% /  1);
```
Comment (bsmth) [inline houdini/css_painting_api/example-boxbg.html:10]: Same here about alpha values of `1`, should we instead use `hsl(55 90% 60%)`?

File: tools/color-mixer/index.html [hunk 4]
```diff
@@ -245,20 +245,20 @@ <h3>Color mixer</h3>
...
-            "rgba(33, 150, 243, 1)",
-            "rgba(3, 169, 244, 1)",
-            "rgba(0, 188, 212, 1)",
-            "rgba(0, 150, 136, 1)",
-            "rgba(76, 175, 80, 1)",
-            "rgba(139, 195, 74, 1)",
-            "rgba(205, 220, 57, 1)",
-            "rgba(255, 235, 59, 1)",
-            "rgba(255, 193, 7, 1)",
+            "rgb(244 67 54 /  1)",
```
Comment (bsmth) [inline tools/color-mixer/index.html:248]: is the alpha channel necessary here?

**Comment by bsmth** [comment]
couple of minor comments, overall looking fine, ty!

File: tools/color-mixer/index.html (hunk 4 above)
Comment (OnkarRuikar) [reply tools/color-mixer/index.html:248]: Nope. Removed.

File: houdini/css_painting_api/example-boxbg.html (hunk 3 above)
Comment (OnkarRuikar) [reply houdini/css_painting_api/example-boxbg.html]: Good catch! I've removed all the opacity `1`  values.

**Review by bsmth — APPROVED** [review]
LGTM, thank you!

**Comment by bsmth** [comment]
@OnkarRuikar are there any prose references / other places that need updating following this merge?

**Comment by OnkarRuikar** [comment]
- @bsmth need to merge https://github.com/mdn/css-examples/pull/196 in the same repo. And https://github.com/mdn/content/pull/35613 in content repo.

**Comment by bsmth** [comment]
>need to merge https://github.com/mdn/css-examples/pull/196 in the same repo

✔️ done, left two comments on the content PR for complete removals

**Pull request merged**